shimming-toolbox/shimmingtoolbox/cli/mask.py

```diff
@@ -303,6 +304,85 @@ def sct(fname_input, fname_output, contrast, centerline, file_centerline, brain,
+@click.option('-o', '--output', 'fname_output', type=click.Path(), default=os.path.join(os.curdir, 'mask.nii.gz'),
+              show_default=True, help="Name of output mask. Supported extensions are .nii or .nii.gz.")
+@click.option('-f', '--f_param', required=False, type=float, default=0.5,
+              help="fractional intensity threshold (0->1); default=0.5; smaller values give larger brain outline estimates")
+@click.option('-g', '--g_param', type=float, required=False, default=0,
+              help="vertical gradient in fractional intensity threshold (-1->1); positive values give larger brain outline at bottom, smaller at top")
+@click.option('-v', '--verbose', type=click.Choice(['info', 'debug']), default='info', help="Be more verbose")
+def bet(fname_input, fname_output, f_param, g_param, verbose):
+
+    set_all_loggers(verbose)
+    # Prepare the output
+    create_output_dir(fname_output, is_file=True)
+
+    # Make sure input path exists
+    if not os.path.exists(fname_input):
+        raise RuntimeError("Input file does not exist")
+
+    # Get the number of dimensions
+    nii_input = nib.load(fname_input)
+    ndim = nii_input.ndim
+    # If 4d, last dimension is time, average last dim for better SNR
+    if ndim == 4:
+        input_3d = np.mean(nii_input.get_fdata(), 3)
+        nii_3d = nib.Nifti1Image(input_3d, affine=nii_input.affine, header=nii_input.header)
+        fname_mean = os.path.join(os.path.dirname(fname_output), 'mean_3d.nii.gz')
+        nib.save(nii_3d, fname_mean)
+        fname_process = fname_mean
+    # If not then only set the processing filename
+    else:
+        fname_process = fname_input
+
+    # Run BET
+    # Create the mask
+    run_subprocess(['bet', fname_process, fname_output, '-f', str(f_param), '-g', str(g_param), '-m'])
+    
+    return fname_output
+
+
+@mask_cli.command(context_settings=CONTEXT_SETTINGS,
+                  help="Wrapper for modify_binary_mask. Lets the user dilate or erode their masks")
+@click.option('-i', '--input', 'fname_input', type=click.Path(), required=True,
+              help="Input path of the nifti file to mask. This nifti file must be 3D. Supported "
+                   "extensions are .nii or .nii.gz.")
+@click.option('-o', '--output', 'fname_output', type=click.Path(exists=True), default=os.path.join(os.curdir, 'mask.nii.gz'),
+              show_default=True, help="Name of output mask. Supported extensions are .nii or .nii.gz.")
+@click.option('--shape', 'shape', required=False, type=click.Choice(['sphere', 'cross', 'line', 'cube', 'None']), default='sphere',
+              help="3d kernel to perform the dilation. Allowed shapes are: 'sphere', 'cross', 'line', 'cube', 'None'.")
+@click.option('--size', 'size', type=float, required=True, default=1,
+              help="Kernel size for the dilation or erosion. Must be odd.")
+@click.option('--operation', 'operation', type=str, required=True, default="dilate",
```

Comment:
Use click.Choice(xxxx), this will remove the need to specify it in the docstring